project.clj

```diff
@@ -7,16 +7,18 @@
                  [org.clojure/tools.cli "0.3.1"]
                  [cheshire "5.3.1"]
                  [clj-aws-s3 "0.3.10"]
-                 [clout "2.1.0"]
-                 [compojure "1.3.2"]
+                 [compojure "1.4.0"]
+                 [consul-clojure "0.1.0-SNAPSHOT"]
```

Comment:
When we do this we are making `vignette` very wikia specific. We are coupling it to our service discovery choice and to the static asset service.